internal/dcache/replication_manager/replication_manager.go

```diff
@@ -864,13 +871,13 @@ func copyOutOfSyncChunks(job *syncJob) error {
 		if info.ModTime().UnixMicro() > job.syncStartTime {
 			// This chunk is created after the sync start time, so it will be written to both source and target
 			// RVs by the client PutChunk() RPC calls, so we can skip it here.
-			log.Debug("ReplicationManager::copyOutOfSyncChunks: Skipping chunk %s/%s, created after sync start time %d",
-				sourceMVPath, entry.Name(), job.syncStartTime)
+			log.Debug("ReplicationManager::copyOutOfSyncChunks: Skipping chunk %s/%s, Mtime (%d) > syncStartTime (%d)",
+				sourceMVPath, entry.Name(), info.ModTime().UnixMicro(), job.syncStartTime)
```

Comment:
instead of printing as usecs, which will be big numbers hard to read and compare, can we log it as time.Duration